specification/schema/tileset.schema.json

```diff
@@ -19,8 +19,54 @@
             "description": "A dictionary object of metadata about per-feature properties.",
             "additionalProperties": {
                 "$ref": "properties.schema.json"
+            },
+            "deprecated": true
+        },
+        "schema": {
+            "allOf": [
+                {
+                    "$ref": "schema.schema.json"
+                }
+            ],
+            "description": "An object defining classes and enums."
+        },
+        "schemaUri": {
+            "type": "string",
+            "description": "The URI (or IRI) of the external schema file.",
+            "format": "iri-reference"
+        },
+        "statistics": {
+            "allOf": [
+                {
+                    "$ref": "statistics.schema.json"
+                }
+            ],
+            "description": "An object containing statistics about entities."
+        },
+        "groups": {
+            "type": "object",
+            "description": "A dictionary, where each key is a group ID and each value is an object defining the group. Group IDs may contain only alphanumeric and underscore characters",
+            "minProperties": 1,
+            "additionalProperties": {
+                "$ref": "group.schema.json"
             }
         },
+        "metadata": {
+            "allOf": [
+                {
+                    "$ref": "metadataEntity.schema.json"
+                }
+            ],
+            "description": "Tileset metadata."            
+        },
+        "contentInfo": {
```

Comment:
`contentInfo` is similar `3DTILES_content_gltf`. The sub-object `contentInfo.gltf` says what glTF extensions are used/required. In the future it might say what profile the glTF contents conform - for example a point cloud profile. 

More sub-objects could be added for other content formats in the future.

Reply:
I'm not really happy with the name `contentInfo`. Suggestions welcome...

Reply:
I removed `contentInfo` for now. Will open up a dedicated issue to discuss this some more.